src/h265_bitstream_parser.cc

```diff
@@ -24,8 +23,6 @@ typedef absl::optional<h265nal::H265NalUnitParser::NalUnitState>
 
 // The size of a full NALU start sequence {0 0 0 1}, used for the first NALU
 // of an access unit, and for SPS and PPS blocks.
-const size_t kNaluLongStartSequenceSize = 4;
```

Comment:
Do you need to remove this const for the package to build? It should be needed somewhere. If it's not, it may be an issue in the code. 

If you really need to remove it, please comment it out instead

Reply:
Could comment this out. The lint in WebRTC is straight and cannot include any unused variables.

Reply:
Same as other variables I deleted.

Reply:
Please comment them out instead of removing them